Cargo.toml

```diff
@@ -9,30 +9,54 @@ repository = "https://github.com/collinsmuriuki/mpesa-rust"
 readme = "./README.md"
 license = "MIT"
 
+[features]
+default = [
+    "account_balance",
+    "b2b",
+    "b2c",
+    "bill_manager",
+    "c2b_register",
+    "c2b_simulate",
+    "express_request",
+    "transaction_reversal",
+    "transaction_status",
```

Comment:
If we end up having `wasm` as a feature flag, would it make more sense to prefix the api specific feature names with `"api/<feature_name>"?